tests/CMakeLists.txt

```diff
@@ -7,52 +7,91 @@ endforeach()
 
 PODIO_GENERATE_DATAMODEL(datamodel datalayout.yaml headers sources)
 
-file(GLOB sources_utils utilities/*.cc)
-file(GLOB headers_utils utilities/*.h)
+if (ENABLE_SIO)
+  # Filter out the sio block handlers to avoid polluting the TestDataModel lib
```

Comment:
The same issue will appear in edm4hep.
Can we find a way to make this automated for users of PODIO_GENERATE_DATAMODEL?

Reply:
Yes, we definitely should. I would suggest to discuss this in #131.